### ISSUE TITLE
bump deadpool-postgres version

### ISSUE THREAD

**Issue by godofdream** [bikeshedder/deadpool#66]
I added deadpool-postgres 0.5.6 which should depend on tokio-postgres=0.6.0
however the crate depends on tokio-postgres= "0.5.1"

maybe bumb deadpool-postgres to 0.6.0 to keep on paar with tokio-postgres and publish a new crate?

**Comment by bikeshedder** [comment]
This has already been done as of #65 and 227d4e4be4cd2a98f3d3a4d8f2f0a1f4263e20ee. I just haven't prepared a release on crates.io, yet. Originally I tried to support both `tokio-postgres 0.5` and `0.6` but due to the `tokio 0.3` changes I decided against this and make this a new `MINOR` release.

I'm still thinking about disabling most of the default features of the dependent crates as proposed by #60. Yet I'm not too happy about proxying all features. So I'm currently thinking about just disabling the default features and putting a big fat warning in the `README` that one needs to make sure to depend on the same version `tokio-postgres` version as `deadpool-postgres` if enabling extra features or it will result in an compile error.

That's something I wanted to figure out before pushing a new version to crates.io as this requires a new `MINOR` version just like bumping the `tokio` and `tokio-postgres` version.

My own project is currently blocked by https://github.com/hyperium/hyper/issues/2302 so I'm not upgrading to `tokio-postgres 0.6` and `tokio 0.3` until this is fixed.

Please give the version in the repository a try. If that works for you I'll publish a new version on crates.io asap.

**Comment by godofdream** [comment]
I switched to master and it compiles. However i have some issues with tokio 0.3 and will give feedback when it works.

**Comment by shiftrtech** [comment]
I stay tune, nice lib

**Comment by jbg** [comment]
master is working well for me with tokio 0.3

**Comment by godofdream** [comment]
Works for me, too.

**Comment by bikeshedder** [comment]
I'll release a new version on `crates.io` soon. It seams there are no issues running `deadpool` with `tokio 0.3`. :tada:

**Comment by bikeshedder** [comment]
I just released `deadpool-postgres 0.6` on crates.io which adds support for `tokio 0.3` and `tokio-postgres 0.6`:

https://crates.io/crates/deadpool-postgres/0.6.0

**Comment by jbg** [comment]
> I'm still thinking about disabling most of the default features of the dependent crates as proposed by #60. Yet I'm not too happy about proxying all features. So I'm currently thinking about just disabling the default features and putting a big fat warning in the `README` that one needs to make sure to depend on the same version `tokio-postgres` version as `deadpool-postgres` if enabling extra features or it will result in an compile error.

fwiw, I've always depended on `deadpool-postgres` in this way, so your suggestion seems like a good solution to me:

```
deadpool-postgres = { version = "0.6.0", default-features = false }
tokio-postgres = { version = "0.6.0", features = ["with-chrono-0_4", "with-serde_json-1", "with-uuid-0_8"] }
```
